### DISCUSSION TITLE
Azure TRE Accelerator Community Meetup - 13 Nov

### DISCUSSION THREAD

**Discussion by CCCowley** [microsoft/AzureTRE#4129] [General]
Following a really successful meetup last month exploring how well the Azure TRE is working in practice, we look forward to bringing the community together again for our next meeting.

In this session, we're going to be focusing on how we can continue to build upon the Azure TRE accelerator to provide the flexibility to support a wide range of use cases, while providing the scalability and stability to enable large-scale, long-term, mission-critical deployments.

Register to join the meetup: https://events.teams.microsoft.com/event/12b68fb2-4e1e-4d20-84f0-355b6567b1ee@1fb496e0-1eea-4fbd-be38-d4b9cda1d791

Look forward to seeing you there!

**Comment by JaimieWi** [comment]
Hi @CCCowley 

Thanks you for hosting this! It was very interesting to hear the different experiences with the TRE. I unfortunately couldn't join from the start and would love to rewatch, I don't have the permissions to the recording. Is it possible to get access to this?

Jaimie - Software Engineer OUH

**Reply by CCCowley** [reply]
Hi @JaimieWi 

I've just uploaded the recording here: https://youtu.be/iLg9BbZ2Eg4?si=iFWJmXZuRX9nNkxI 

Thanks,

Craig

**Comment by CCCowley** [comment]
Hi everyone,

Thank you for joining the community meetup this week. If you missed the session or just want to rewatch, the recording is available here: https://youtu.be/iLg9BbZ2Eg4?si=iFWJmXZuRX9nNkxI

Thanks.

Craig
